user/models.py

```diff
@@ -1,10 +1,23 @@
-from django.db import models
 from django.contrib.auth.models import AbstractUser
+from django.db import models
 from phonenumber_field.modelfields import PhoneNumberField
 
+def directory_path(instance, filename):
+    return 'profile/images/user_{0}/{1}'.format(instance.id, filename)
```

Comment:
```suggestion
    return f'profile/images/user_{instance.id}/{filename}'
```
最近のpythonだとあまり`.format()`は見ないので，理由がなければ文字列リテラルがおすすめです